src/BloomExe/WebLibraryIntegration/BookTransfer.cs

```diff
@@ -203,11 +196,11 @@ internal void HandleBloomBookOrder(string argument)
 				if (_progressDialog.ProgressStateResult != null &&
 					_progressDialog.ProgressStateResult.ExceptionThatWasEncountered != null)
 				{
-					Palaso.Reporting.ErrorReport.ReportFatalException(
-						_progressDialog.ProgressStateResult.ExceptionThatWasEncountered);
+						Palaso.Reporting.ErrorReport.ReportFatalException(
+							_progressDialog.ProgressStateResult.ExceptionThatWasEncountered);
+					}
```

Comment:
This appears to be only a whitespace change.  (which I'm not complaining about, mind you)